packages/mobile/src/store/shelves/sagas/shelfFlowerAdd.saga.ts

```diff
@@ -16,7 +16,6 @@ function* handle(action: ShelfFlowerAdd) {
       body: action.payload,
     });
 
-    console.dir(result);
     if (result.status === 403) {
```

Comment:
I would suggest moving status codes to some reuseable constants

Reply:
Thanks, I will fix it.